prow/hook/events.go

```diff
@@ -171,3 +178,12 @@ func (s *Server) handleStatusEvent(se github.StatusEvent) {
 		}(p, h)
 	}
 }
+
+func (s *Server) pluginClientFor(org, repo string) plugins.PluginClient {
+	client := s.Plugins.PluginClient
+	if trigger := s.Plugins.TriggerFor(org, repo); trigger != nil && trigger.TrustedOrg != "" {
```

Comment:
See my above comment on that... not happy about it either: https://github.com/kubernetes/test-infra/pull/3790#discussion_r130691307